pkgs/by-name/te/temporal-cli/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  installShellFiles,
+}:
+
+buildGoModule rec {
+  pname = "temporal-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "temporalio";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-y0C2z2iMMQSG5+xGngZ98+ixIgbvaQxPdAWuPbEbBAY=";
+  };
+
+  vendorHash = "sha256-zhGqDHdVGg7eGnw5L3eSyXKBTjp85ir5zrtf7HbXmC0=";
+
+  overrideModAttrs = old: {
+    # https://gitlab.com/cznic/libc/-/merge_requests/10
```

Comment:
Why don't we fetchpatch this file?

Reply:
I'm not very sure. But it seems to come from https://github.com/NixOS/nixpkgs/pull/242721. 

It seems like a patch for vendoring .